starter-code/server.js

```diff
@@ -172,7 +173,7 @@ function loadArticles() {
 
 function loadDB() {
   // COMMENT: What number(s) of the full-stack-diagram.png image correspond to the following line of code? Which method of article.js is interacting with this particular piece of `server.js`? What part of CRUD is being enacted/managed by this particular piece of code?
-  // Put your response here...
+  // RESPONSE: ? (I'm actually confused between loadArticles() and loadDB()) / Article.prototype.insertRecord / Update
```

Comment:
2, 3, 4, 5 / Article.prototype.insertRecord / Create

Sorry you were out sick on this one! If you want to go over any of the code just snag me or Devon at the next lab - we'd be happy to review it with you!